Students/Sebastian/CommandBase/commands/DriveTrainCommand.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.templates.commandbased.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import Assignments.Students.Sebastian.CommandBase.subsystem.DriveTrainSub;
+/** An example command that uses an example subsystem. */
+public class DriveTrainCommand extends CommandBase {
+  private final ExampleSubsystem m_subsystem;
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  public ExampleCommand(ExampleSubsystem subsystem) {
+    m_subsystem = subsystem;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {}
```

Comment:
Your DriveTrainCommand execute method is empty. This is the method that you call in Robot.java to tell the RIO which subsystem methods to execute. Currently this program does nothing. Add the relevant methods from your drive train subsystem to this method call.